tuftool/tests/create_repository_integration.rs

```diff
@@ -0,0 +1,246 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+mod test_utils;
+use assert_cmd::Command;
+use chrono::{Duration, Utc};
+use std::env;
+use std::fs::File;
+use tempfile::TempDir;
+use tough::{ExpirationEnforcement, Limits, Repository, Settings};
+
+// This file include integration tests for KeySources: tough-ssm, tough-kms and local file key.
+// Since the tests are run using the actual "AWS SSM and AWS KMS", you would have to configure
+// AWS credentials with root permission.
+// Refer https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html to configure named profile.
+// Additionally, tough-kms key generation is not supported (issue  #211), so you would have to manually create kms CMK key.
+// To run test include feature flag 'integ' like : "cargo test --features=integ"
+
+fn get_profile() -> String {
+    match env::var("AWS_PROFILE") {
+        Ok(value) => value,
+        Err(_e) => String::from("default"),
+    }
```

Comment:
If we provide an empty string for the profile, it will use the default credentials provider, which permits usage of environment/IMDS credentials and the like. (And the `Default` impl for String is an empty string, so...)

```suggestion
    env::var("AWS_PROFILE").unwrap_or_default()
```

Reply:
That was way simplified 👍 . Done.